projects/core/interfaces/data-list-host.ts

```diff
@@ -2,8 +2,8 @@ import {TuiIdentityMatcher, TuiStringHandler} from '@taiga-ui/cdk';
 
 // TODO: Consider refactoring checkOption, it is only needed in ComboBox
 export interface TuiDataListHost<T> {
-    checkOption?(option: T): void;
-    handleOption(option: T): void;
+    checkOption?(option: T, element: HTMLElement): void;
```

Comment:
Let's not expand the public API and instead reuse select option component that does the scroll automatically.

Reply:
fixed